app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -131,6 +145,14 @@ class ExplorationActivityPresenter @Inject constructor(
     return ephemeralStateResult.getOrDefault(Exploration.getDefaultInstance())
   }
 
+  private fun backPressActivitySelector(backflowScreen: Int?){
+    when(backflowScreen){
+      ParentActivityForExploration.BACKFLOW_SCREEN_STORY.value -> activity.startActivity(StoryActivity.createStoryActivityIntent(context,internalProfileId,topicId,storyId))
```

Comment:
Spacing issue: `createStoryActivityIntent(context, internalProfileId, topicId, storyId)`

Reply:
Ditto for below two lines, there should be space after `,`

Reply:
Done